docs/source/zero-code-change.rst

```diff
@@ -186,10 +186,10 @@ For Jupyter notebooks, use the following approach to turn on both:
 .. code-block::
 
    %load_ext cudf.pandas
-   from cuml.experimental.accel import install
+   from cuml.accel import install
    install()
 
-A single magic invocation will be available in the next version of cuML.
+A single magic invocation is planned for a future release of cuML.
```

Comment:
Do we want to say something even that strong? I'm loathe to promise things in docs - personally I'd drop this entirely, or maybe hedge more like

```suggestion
A single magic invocation may be possible in a future release of cuML.
```

or maybe say "future release of RAPIDS" since we might want to unify the magics used here.

Reply:
I'd remove the whole sentence about "single magic" and what might be in the furture. It isn't actionable for the user.

Using two `%load_ext` lines is super good enough and simple to understand. Users who learnt how to use cudf.pandas don't have to relearn how to use it when they want to start using cul.accel (and vice versa).

The important thing is that they actually compose and enabling both at the same time (in either order) doesn't break things.

Reply:
I'll fully remove that.

Reply:
Done in fab415e0731c3cce203757e47a0c3eda992246f1 .